src/m365/cli/commands/config/config-set.ts

```diff
@@ -0,0 +1,77 @@
+import * as chalk from "chalk";
+import { Logger } from "../../../../cli";
+import { CommandOption } from "../../../../Command";
+import configstore from "../../../../configstore";
+import GlobalOptions from "../../../../GlobalOptions";
+import AnonymousCommand from "../../../base/AnonymousCommand";
+import commands from "../../commands";
+const Configstore = require('configstore');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  key: string;
+  value: string;
+}
+
+class CliConfigSetCommand extends AnonymousCommand {
+
+  public get name(): string {
+    return `${commands.CONFIG_SET}`;
+  }
+
+  public get description(): string {
+    return 'Manage global configuration settings about the CLI for Microsoft 365';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
+    let value: any = '';
```

Comment:
Shouldn't we set the default value to `undefined`? Currently, we could get into a situation where we set a string and we expect a boolean.